src/gobstuf/config.py

```diff
@@ -27,3 +27,5 @@ def _getenv(varname, default_value=None, is_optional=False):
 # Optional parameters
 PKCS12_FILENAME = _getenv("PKCS12_FILENAME", is_optional=True)
 PKCS12_PASSWORD = _getenv("PKCS12_PASSWORD", is_optional=True)
+
+API_BASE_PATH = _getenv("BASE_PATH", default_value="/gob_stuf")
```

Comment:
Propose to have this value default empty. Deployment will set it to a value that matches the deployed API